versions-common/src/main/java/org/codehaus/mojo/versions/api/Property.java

```diff
@@ -176,4 +184,14 @@ public void setBanSnapshots( boolean banSnapshots )
     {
         this.banSnapshots = banSnapshots;
     }
+
+    public String getValue()
```

Comment:
The original idea behind omitting the value in comparison was that properties with different values were treated as the same property. That's why the Property class didn't have a value property.

Reply:
[See above](#discussion_r1038304268-permalink)